### PR TITLE
fix default value

### PR DESCRIPTION
**Pull request by frankleaf**
### Ⅱ. Does this pull request fix one issue?

fixes #XXXX

### Ⅴ. Special notes for reviews

### CODE REVIEW

**Comment by cheyang** [comment]
@frankleaf Please fix the ut. Thanks.

**Comment by frankleaf** [comment]
> @frankleaf Please fix the ut. Thanks.
done

File: pkg/ddc/jindocache/transform.go [hunk 1]
```diff
@@ -275,17 +275,22 @@ func (e *JindoCacheEngine) transformMaster(runtime *datav1alpha1.JindoRuntime, m
...
 
-		if mount.Options["cacheStrategy"] == "DHT" && mount.Options["metaPolicy"] == "ONCE" {
-			cacheStrategy = mount.Options["cacheStrategy"]
+		if mount.Options["metaPolicy"] == "ALWAYS" {
 			metaPolicy = mount.Options["metaPolicy"]
 		}
 
-		if mount.Options["metaPolicy"] == "ONCE" {
-			metaPolicy = mount.Options["metaPolicy"]
+		if mount.Options["cacheStrategy"] == "DHT" && metaPolicy == "ONCE" {
```
Comment (cheyang) [inline pkg/ddc/jindocache/transform.go:286]: There are two checks for mount.Options["cacheStrategy"] == "DHT" in the code. How about using the following way:

```golang
if cacheStrategy := mount.Options["cacheStrategy"]; cacheStrategy == "DHT" {
    if metaPolicy == "ONCE" {
        cacheStrategy = cacheStrategy 
    } else {
        err = fmt.Errorf("CacheStrategy DHT must be used with metaPolicy ONCE and current metaPolicy is %s", metaPolicy)
        e.Log.Error(err, "Incorrect metaPolicy", metaPolicy)
        return err
    }
}
```

**Review by cheyang — CHANGES_REQUESTED** [review]

File: pkg/ddc/jindocache/transform.go [hunk 2]
```diff
@@ -275,17 +275,22 @@ func (e *JindoCacheEngine) transformMaster(runtime *datav1alpha1.JindoRuntime, m
...
 		}
 		cacheStrategy := "DISTRIBUTED"
-		metaPolicy := "ALWAYS"
+		metaPolicy := "ONCE"
 		readCacheReplica := 1
 		writeCacheReplica := 1
 
-		if mount.Options["cacheStrategy"] == "DHT" && mount.Options["metaPolicy"] == "ONCE" {
-			cacheStrategy = mount.Options["cacheStrategy"]
+		if mount.Options["metaPolicy"] == "ALWAYS" {
```
Comment (cheyang) [inline pkg/ddc/jindocache/transform.go:282]: For metaPolicy, it only checks for the "ALWAYS" case but not the "ONCE" case. You might want to add a more thorough check.
```golang
if userMetaPolicy, ok := mount.Options["metaPolicy"]; ok {
    if userMetaPolicy == "ONCE" || userMetaPolicy == "ALWAYS" {
         metaPolicy = userMetaPolicy
    } else {
         err = fmt.Errorf("Invalid metaPolicy: %s", userMetaPolicy)
         e.Log.Error(err, "Invalid metaPolicy")
         return err
    }
}
```

**Review by cheyang — CHANGES_REQUESTED** [review]

**Review by cheyang — APPROVED** [review]
/lgtm
/approve

**Pull request merged**